samples/cellular/nrf_cloud_mqtt_fota/README.rst

```diff
@@ -0,0 +1,178 @@
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+The sample requires an `nRF Cloud`_ account.
+
+Your device must be onboarded to nRF Cloud.
+If it is not, follow the instructions in `Device onboarding <nrf_cloud_mqtt_fota_sample_onboarding_>`_.
+
+.. note::
+   This sample requires modem firmware v1.3.x or later for an nRF9160 SiP and v2.0.0 or later for nRF9161 and nRF9151 SiPs.
+
+.. include:: /includes/external_flash_nrf91.txt
+
+.. note::
+   Full modem FOTA requires development kit version 0.14.0 or higher if you are using an nRF9160 DK.
+
+Overview
+********
+
+You can update your device firmware on the `nRF Cloud`_ portal or directly through the `nRF Cloud MQTT API`_.
+See the `nRF Cloud Getting Started FOTA documentation`_ for details.
+
+
+User interface
+**************
+
+Button 1:
+   Check for a FOTA update right away without waiting for the timeout.
+
+.. _nrf_cloud_mqtt_fota_sample_onboarding:
+
+Setup
+=====
+
+You must onboard your device to nRF Cloud for this sample to function.
+You only need to do this once for each device.
+
+To onboard your device, install `nRF Cloud Utils`_ and follow the instructions in the README.
+
+Configuration
+*************
+|config|
+
+Configuration options
+=====================
+
+Check and configure the following configuration options for the sample:
+
+.. _CONFIG_CLOUD_CONNECTION_RETRY_TIMEOUT_SECONDS:
+
+CONFIG_CLOUD_CONNECTION_RETRY_TIMEOUT_SECONDS - Cloud connection retry timeout (seconds)
+   Sets the cloud connection retry timeout in seconds.
+
+.. _CONFIG_CLOUD_READY_TIMEOUT_SECONDS:
+
+CONFIG_CLOUD_READY_TIMEOUT_SECONDS - Cloud readiness timeout (seconds)
+   Sets the cloud readiness timeout in seconds.
+   If the connection to nRF Cloud does not become ready within this timeout, the sample will reset its connection and try again.
+
+.. include:: /libraries/modem/nrf_modem_lib/nrf_modem_lib_trace.rst
+   :start-after: modem_lib_sending_traces_UART_start
+   :end-before: modem_lib_sending_traces_UART_end
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/cellular/nrf_cloud_mqtt_fota`
+
+.. include:: /includes/build_and_run_ns.txt
+
+The configuration files for this sample are located in the :file:`samples/cellular/nrf_cloud_mqtt_fota` folder.
+See :ref:`configure_application` on how to configure the parameters.
+
+To create a FOTA test version of this sample, change the ``PATCHLEVEL`` in the :file:`VERSION` file.
+
+To enable full modem FOTA, add the following parameter to your build command:
+
+``-DEXTRA_CONF_FILE=full_modem_fota.conf``
+
+Also, if you are using an nRF9160 DK, specify your development kit version by appending it to the board name.
+For example, if you are using version 1.0.1, use the board name ``nrf9160dk@1.0.1/nrf9160/ns`` in your build command.
```

Comment:
version 1.0.1? There is no such board target https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nordic/nrf9160dk/board.yml also this is a board target not a board/board name